source/mainnet/net/guides/company-identities.rst

```diff
@@ -34,17 +39,26 @@ The information below describes how to create a company identity. Note that the
+         user_cli generate-request-v1 --cryptographic-parameters cryptographic-parameters.json \
+                          --ars ars.json \
+                          --ip-info ip-info.json \
+                          --request-out request.json # request to send to the identity provider
 
-   #. To create additional accounts from the identity object returned by Notabene, follow the `create accounts instructions <https://github.com/Concordium/concordium-base/blob/main/rust-bins/docs/user-cli.md#create-accounts-from-a-version-1-identity-object>`__. You must deploy the credential.json output file to the chain exactly as described. If you don't, the account will not be created. You need access to a node to complete this step. Store the auxiliary output securely.
+      The above command will ask for some additional input. You have to choose anonymity revokers and revocation threshold. Use arrow keys to navigate through the lists and the space key to select and deselect list entries. Select whether the identity shall be used for Mainnet or Testnet. Afterwards, 24 BIP-39 will be generated and shown; write down the words and type them in again. This is your secret recovery phrase.
 
-   #. To recover your identity object (e.g. if you lost it), follow the `recovery of identity instructions <https://github.com/Concordium/concordium-base/blob/main/rust-bins/docs/user-cli.md#recovery-of-identity>`__. Email the ``recovery-request.json`` output file to ania@notabene.id. When the recovery request has been validated successfully, Notabene will notify you by email, and they will return the identity object named ``id-object.json`` that you lost. With the recovered identity object, you can then recreate your account keys, if needed.
+      The command outputs the ``request.json`` file which contains the request that should be sent to ania@notabene.id. Store the auxiliary output securely.
 
-   If you experience issues with steps 1, 2, 3, 4, 6 or 7, please contact Concordium’s technical support via support@concordium.software. If you experience issues with step 5, identity verification, please contact Notabene via ania@notabene.id.
+   #. To verify your identity towards Notabene, follow the `entity verification instructions <https://notaben.notion.site/Entity-verification-next-1b4fbcd8e32042e1ac3b0018a3cc27bc>`_. When the identity has been verified successfully, Notabene will notify you by email, and they will send you an identity object file named ``id-object.json``. If you experience any issues with this step, identity verification, please contact Notabene via ania@notabene.id.
+
+   If you experience issues with steps 1, 2, 3, or 4 please contact Concordium’s technical support via support@concordium.software.
 
 
 .. dropdown:: Testnet
```

Comment:
I have uploaded new bundles with testnet files.

Can you please update the testnet links and the checksums?

- https://distribution.concordium.software/tools/linux/enterprise-identities-v2-testnet.tar.gz
- https://distribution.concordium.software/tools/windows/signed/enterprise-identities-v2-testnet.zip
- https://distribution.concordium.software/tools/macos/signed/enterprise-identities-v2-testnet.zip